script.js

```diff
@@ -0,0 +1,70 @@
+const add = document.querySelector('.button');
+const remove = document.querySelector('.close');
+const books = JSON.parse(localStorage.getItem('new-list')) || [
+  {
+    title: 'Faust',
+    author: 'Goethe',
+  },
+  {
+    title: 'For Whom the Bell Tolls',
+    author: 'Ernest Hemingway',
+  },
+  {
+    title: 'Old Man and the Sea',
+    author: 'Ernest Hemingway',
+  },
+  {
+    title: 'Freedom at Midnight',
+    author: 'Collins',
+  },
+  {
+    title: 'Jungle Book',
+    author: 'Ruskin Bond',
+  },
+  {
+    title: 'Illiad',
+    author: 'Homer',
+  },
+];
+
+// When some1 hits add
+
+function addFunction(e) {
+  e.preventDefault();
+  const title = document.querySelector('.title');
+  const author = document.querySelector('.author');
+  const list = document.querySelector('#list');
+
+  const newbook = {
+    title: title.value,
+    author: author.value,
+  };
+
+  books.push(newbook);
+  /// ///////// display all books
+  /// / first reset list
+  // put it to locall server
+
+  localStorage.setItem('new-list', JSON.stringify(books));
```

Comment:
- [ ] It would be better if we implement local storage full functionality, data is not preserved in the browser as we can see in the browser console below, once the browser is refreshed, everything is lost in the web page
![Screenshot from 2023-04-24 17-07-20](https://user-images.githubusercontent.com/46388113/234065982-62b6ca89-c7fd-4176-a985-62038f61e6be.png)
![Screenshot from 2023-04-24 17-07-44](https://user-images.githubusercontent.com/46388113/234065975-4fa6a13c-fb9c-4c92-9f90-9b7e4e431a62.png)